.github/actions/start-aws-runner/action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
     required: true
   ec2-image-id:
     # github-self-hosted-runner-ubuntu-20-100g-disk-with-cypress-deps
-    default: "ami-08927c058921b27f4"
+    default: "ami-0f23be2f917510c26"
```

Comment:
just for my own understanding, how do we build new images?
For example do we use packer or something similar, or do we just spin up a new ubuntu instance and make changes to that, or something else?

In general is there some way for us to see what changes are included in these images (other than asking whoever created the PR)?

Reply:
For now we are spin up a new ubuntu instance and make changes.

There is no way now to view changes that was made on AMI. I think packer is a good solution to do that, but it requires investigation. We need all changes that was applied on this image or create it from scratch.

Reply:
got it, thanks!

Reply:
@git-phu https://github.com/airbytehq/airbyte-cloud/blob/master/docs/Things-To-Know/Builds/Builds.md

Reply:
on the packer front - it's a trade off between making it accessible to everyone and having to bring on something more specialised. maybe packer is worth it.

Reply:
agreed.  I think for me the value to something like packer is it essentially documents what is in each ami.  Otherwise, I'm not sure we have a good way to keep track of that at current.

Reply:
an interesting thought I had is what if we used docker instead of packer. it's much more accessible vs packer. but maybe there are too many downside. we can discuss somewhere else!